docs/source/_redirects

```diff
@@ -1,15 +1,13 @@
-/federation-2/*  /docs/federation/v2/federation-2/:splat
+/federation-2/coming-to-federation-2/ /docs/federation/v2/federation-2/under-consideration/
```

Comment:
We can actually cut these first two (I've done so in a separate PR that hasn't landed yet)

Reply:
Let's just let your PR remove them separately for hopefully fewer merge conflicts.